### PR TITLE
Approve only button

### PR DESCRIPTION
**Pull request by sampettersson**
## [APP-539]

- Fix approve only button not showing

## Checklist

- [x] Dark/light mode verification
- [x] Landscape verification
- [x] iPad verification
- [x] iPod/iPhone 12 mini/iPhone SE verification

[APP-539]: https://hedvig.atlassian.net/browse/APP-539

### CODE REVIEW

File: Projects/Offer/Sources/Offer.swift [hunk 1]
```diff
@@ -209,7 +209,11 @@ class OfferState {
 		return client.perform(mutation: GraphQL.SignQuotesMutation(ids: ids))
 			.map { data in
 				if data.signQuotes.asAlreadyCompleted != nil {
-					self.$hasSignedQuotes.value = true
+					Future().delay(by: 1)
```
Comment (tarikstafford) [inline Projects/Offer/Sources/Offer.swift:212]: What's the delay for here?

**Review by tarikstafford — CHANGES_REQUESTED** [review]

File: Projects/Offer/Sources/Offer.swift (hunk 1 above)
Comment (sampettersson) [reply Projects/Offer/Sources/Offer.swift:212]: Because if signing was too fast it felt like something went wrong 😅

Comment (sampettersson) [reply Projects/Offer/Sources/Offer.swift:212]: Buut, thinking about it signing in prod might be slow enough already 🤔

Comment (tarikstafford) [reply Projects/Offer/Sources/Offer.swift:212]: Can't this introduce race conditions tho?

Comment (tarikstafford) [reply Projects/Offer/Sources/Offer.swift:212]: Yeah I don't think we should introduce that much delay tho if we don't need it. Maybe 0.5 is ok.

**Review by tarikstafford — APPROVED** [review]

**Review by tarikstafford — APPROVED** [review]

**Pull request merged**